cmd/createcluster.go

```diff
@@ -153,6 +158,7 @@ func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.UintVar(&config.TargetGasLimit, "target-gas-limit", 36000000, "Preferred target gas limit for transactions.")
 	flags.BoolVar(&config.Compounding, "compounding", false, "Enable compounding rewards for validators by using 0x02 withdrawal credentials.")
 	flags.StringVar(&config.ExecutionEngineAddr, "execution-client-rpc-endpoint", "", "The address of the execution engine JSON-RPC API.")
+	flags.StringVar(&config.Zipped, "zipped", "", "Zip created files and folders into a zip or tar file")
```

Comment:
Missing dot at the end. Unclear what files and folder are meant here and what are the possible values.